core/src/scopyaboutpage.cpp

```diff
@@ -17,24 +17,64 @@ QWidget* ScopyAboutPage::buildPage(QString src, QTextDocument::ResourceType r) {
 	QVBoxLayout *lay = new QVBoxLayout(page);
 
 	QTextBrowser *browser = new QTextBrowser(page);
+
+	HoverWidget* hover = new HoverWidget(createNavigationWidget(browser), browser, browser);
+	hover->setAnchorPos(HoverPosition::HP_TOPRIGHT);
+	hover->setContentPos(HoverPosition::HP_BOTTOMLEFT);
+	hover->setAnchorOffset(QPoint(-10, 0));
+	hover->show();
+
 	lay->addWidget(browser);
-	switch(r) {
-	case QTextDocument::MarkdownResource:
-		browser->setMarkdown(src);
-		break;
-	case QTextDocument::UnknownResource:
-	case QTextDocument::HtmlResource:
-	default:
-		browser->setSource(src,r);
-		break;
-	}
-	QPushButton *backButton = new QPushButton("Back",page);
-	backButton->setProperty("blue_button",true);
-	lay->addWidget(backButton);
-	connect(backButton,SIGNAL(clicked()),browser,SLOT(backward()));
+	browser->setSource(src,r);
```

Comment:
Please review this
This implementation requires that the plugin developer creates an explicit "about" file, and cannot rely on simply returning a string.